src/D2L.CodeStyle.Analyzers/CommonFixes/AddAttributeCodeFix.cs

```diff
@@ -135,18 +145,21 @@ string attrName
 			// add an attribute list.
 			decl = decl.WithoutLeadingTrivia();
 
-			// TypeDeclarationSyntax (the base class of these 3 types) doesn't
-			// have AddAttributeLists... so we need to copy+paste some code.
+			// MethodDeclarationSyntax doesn't have AddAttributeLists... so we

```

Comment:
```suggestion
			// MemberDeclarationSyntax doesn't have AddAttributeLists... so we
```

Reply:
But also, uh, https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.syntax.memberdeclarationsyntax.addattributelists?view=roslyn-dotnet-4.0.1#microsoft-codeanalysis-csharp-syntax-memberdeclarationsyntax-addattributelists(microsoft-codeanalysis-csharp-syntax-attributelistsyntax())

I think this code section is out of date?

Reply:
It might be, I can check after.